homeassistant/components/unifiprotect/const.py

```diff
@@ -41,11 +45,24 @@
 MIN_REQUIRED_PROTECT_V = Version("1.20.0")
 OUTDATED_LOG_MESSAGE = "You are running v%s of UniFi Protect. Minimum required version is v%s. Please upgrade UniFi Protect and then retry"
 
+SERVICE_SET_DOORBELL_MESSAGE = "set_doorbell_message"
+
+TYPE_EMPTY_VALUE = ""
+
 PLATFORMS = [
     Platform.BUTTON,
     Platform.CAMERA,
     Platform.LIGHT,
     Platform.MEDIA_PLAYER,
     Platform.NUMBER,
+    Platform.SELECT,
     Platform.SWITCH,
 ]
+
+SET_DOORBELL_LCD_MESSAGE_SCHEMA = vol.Schema(
+    {
+        vol.Required(ATTR_ENTITY_ID): cv.entity_ids,
+        vol.Required(ATTR_MESSAGE): cv.string,
+        vol.Optional(ATTR_DURATION, default="None"): cv.string,
```

Comment:
This is a bit odd that it takes `"None"` here? Can we get rid of that?

Reply:
If you make this a `dict`, it should take care of adding `ATTR_ENTITY_ID`

Reply:
https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity_platform.py#L687

Reply:
Also that gets us area and device mapping support for free
https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/config_validation.py#L971